flash_examples/audio_source_separation.py

```diff
@@ -0,0 +1,42 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import flash
+from flash.audio import AudioSourceSeparationData, AudioSourceSeparator
+from flash.core.data.utils import download_data
+
+# 1. Create the DataModule
+download_data("https://pl-flash-data.s3.amazonaws.com/hymenoptera_data.zip", "./data")  # To-Do : Add a right path
```

Comment:
If you dataset is ready, we can upload it on s3 and it can be downloaded. If it contains a license, let's add it to the .zip file :)

Reply:
@tchaton 
You can download simply by typing
```python
from torch import hub
MINI_URL = "https://zenodo.org/record/3871592/files/MiniLibriMix.zip?download=1"
hub.download_url_to_file(MINI_URL, 'MiniLibriMix.zip')
```
``611MB``
Please upload it on s3 and let me know the link.